examples/with-temporal/pages/api/orders/index.ts

```diff
@@ -0,0 +1,29 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { Connection, WorkflowClient } from '@temporalio/client'
+import { Order } from '../../../temporal/src/interfaces/workflows'
+
+export type Data = {
+  result: string
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  if (req.method !== 'POST') {
+    res.send({ result: 'Error code 405: use POST' })
+    return
+  }
+
+  const userId: string = req.headers.authorization // FIXME insecure 😄
```

Comment:
I still don't love the idea of adding an example that sets the user up to do something insecure - can we improve this somehow?

Reply:
Makes sense! Did this: 8f8e3d7